examples/golang-todo-rest-crud/docker-compose.yml

```diff
@@ -0,0 +1,69 @@
+version: "3.8"
+
+services:
+  mongo:
+    container_name: mongodb
+    image: mongo:5.0.9
+    restart: unless-stopped
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: ${MONGO_USERNAME}
+      MONGO_INITDB_ROOT_PASSWORD: ${MONGO_PASSWORD}
+    ports:
+      - 27017:27017
+    volumes:
+      - ./internal/platform/mongo/initdb.d:/docker-entrypoint-initdb.d:ro
```

Comment:
I think it will be more declarative if we move `./internal/platform/mongo/initdb.d` to `docker/mongo/initdb.d`

Reply:
✔️ Done!